src/main/java/frc/robot/commands/ArmJointControlCommand.java

```diff
@@ -0,0 +1,71 @@
+package frc.robot.commands;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.controller.ArmFeedforward;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.RobotMap;
+import frc.robot.subsystems.ArmJointSystem;
+
+import java.util.PrimitiveIterator;
+
+public class ArmJointControlCommand extends Command {
+    private final ArmJointSystem sub;
+    private double targetPosition;
+    private boolean isInTarget;
+    private boolean hasNewPosition;
+    private TrapezoidProfile motionProfile;
+    private TrapezoidProfile.State motionProfileGoal;
+    private TrapezoidProfile.State motionProfileSetPoint;
+
+    public ArmJointControlCommand(ArmJointSystem sub) {
+        this.sub = sub;
+
+        addRequirements(sub);
+    }
+
+    @Override
+    public void initialize() {
+        isInTarget = false;
+        hasNewPosition = true;
+        targetPosition = 0;
+    }
+
+    @Override
+    public void execute() {
+        if (hasNewPosition) {
+           sub.resetPID();
```

Comment:
There is no "reset PID" for Spark closed loop controller. Additionally, the way you implemented this is weird and problematic, since you basically set the PID to run to a specific setpoint. For what reason? 

Remove